src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -14,7 +14,7 @@ public ArrayList() {
 
     public ArrayList(int initCapacity) {
         if (initCapacity <= ZERO) {
-            throw new ArrayListIndexOutOfBoundsException("Illegal argument to create array");
+            throw new ArrayListIndexOutOfBoundsException("Illegal argument to create array " + initCapacity);
```

Comment:
The line is longer than 100 characters. Please run maven checkstyle plugin locally before pushing